src/Microsoft.Framework.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -14,7 +14,13 @@ public class DirectoryInfoWrapper : DirectoryInfoBase
 
         public DirectoryInfoWrapper(DirectoryInfo directoryInfo, bool isParentPath = false)
         {
-            _directoryInfo = directoryInfo;
+            // Work around for https://github.com/dotnet/corefx/issues/616: DirectoryInfo.Exists returns 
```

Comment:
@troydai - the bug seems to be fixed. https://github.com/dotnet/corefx/issues/616. Do we need this fix? 

>stephentoub commented 3 days ago
Fixed in #640